iterative/src/robot.py

```diff
@@ -9,7 +9,7 @@ class MyRobot(wpilib.IterativeRobot):
     def robotInit(self):
         """Robot-wide initialization code should go here"""
 
-        self.lstick = wpilib.Joystick(1)
+        self.lstick = wpilib.Joystick(0)
```

Comment:
... any particular reason to keep the iterative example? My understanding is that it's been removed upstream.